prow/cmd/deck/job_history.go

```diff
@@ -106,9 +106,9 @@ type jobHistoryTemplate struct {
 
 func (bucket blobStorageBucket) readObject(ctx context.Context, key string) ([]byte, error) {
 	u := url.URL{
-		Scheme:      bucket.storageProvider,
-		Host:        bucket.name,
-		Path:        key,
+		Scheme: bucket.storageProvider,
```

Comment:
Is this a Go 1.18 thing? Surprised the jobs didn't catch it earlier

Reply:
Done